this will fix some not founded path,
specially to tcmplex-panteltje

Joerg Bornkessel <hd_brummy@gentoo.org> 2006 Jul 24
diff -Naur vdrconvert.orig/bin/vdrsync-0.1.2.2-dev2.pl vdrconvert/bin/vdrsync-0.1.2.2-dev2.pl
--- vdrconvert.orig/bin/vdrsync-0.1.2.2-dev2.pl	2006-06-28 15:54:57.223613750 +0200
+++ vdrconvert/bin/vdrsync-0.1.2.2-dev2.pl	2006-06-28 15:57:49.814400000 +0200
@@ -34,17 +34,17 @@
 # Hash that lists potential helper Programs
 #########################################################################
 my %progs = (
-    tcmplex           => "",
-    mplex             => "",
-    tcmplex_panteltje => "",
-    dvdauthor         => "",
-    mkisofs           => "",
-    transcode         => "",
-    tcrequant         => "",
-    requant           => "",
-    "dvd-menu.pl"     => "",
-    "vdrsync-buffer.pl"=>"",
-    "vdrsync.pl"      =>"",
+    tcmplex           => "/usr/bin/tcmplex",
+    mplex             => "/usr/bin/mplex",
+    tcmplex_panteltje => "/usr/bin/tcmplex-panteltje",
+    dvdauthor         => "/usr/bin/dvdauthor",
+    mkisofs           => "/usr/bin/mkisofs",
+    transcode         => "/usr/bin/transcode",
+    tcrequant         => "/usr/bin/tcrequant",
+    requant           => "/usr/bin/requant",
+    "dvd-menu.pl"     => "/usr/bin/dvd-menu.pl",
+    "vdrsync-buffer.pl"=>"/usr/bin/vdrsync-buffer.pl",
+    "vdrsync.pl"      =>"/usr/bin/vdrsync.pl",
 );
 
 
diff -Naur vdrconvert.orig/bin/vdrsync-0.1.3.pl vdrconvert/bin/vdrsync-0.1.3.pl
--- vdrconvert.orig/bin/vdrsync-0.1.3.pl	2006-06-28 15:54:57.219613500 +0200
+++ vdrconvert/bin/vdrsync-0.1.3.pl	2006-06-28 16:02:16.099041750 +0200
@@ -37,19 +37,19 @@
 # Hash that lists potential helper Programs
 #########################################################################
 my %progs = (
-    tcmplex           => "",
-    mplex             => "",
-    tcmplex_panteltje => "",
-    dvdauthor         => "",
-    mkisofs           => "",
-    transcode         => "",
-    tcrequant         => "",
-    requant           => "",
-    mpg123            => "",
-    mp2enc            => "",
-    "dvd-menu.pl"     => "",
-    "vdrsync-buffer.pl"=>"",
-    "vdrsync.pl"      =>"",
+    tcmplex           => "/usr/bin/tcmplex",
+    mplex             => "/usr/bin/mplex",
+    tcmplex_panteltje => "/usr/bin/tcmplex-panteltje",
+    dvdauthor         => "/usr/bin/dvdauthor",
+    mkisofs           => "/usr/bin/mkisofs",
+    transcode         => "/usr/bin/transcode",
+    tcrequant         => "/usr/bin/tcrequant",
+    requant           => "/usr/bin/requant",
+    mpg123            => "/usr/bin/mpeg123",
+    mp2enc            => "/usr/bin/mp2enc",
+    "dvd-menu.pl"     => "/usr/bin/dvd-menu.pl",
+    "vdrsync-buffer.pl"=>"/usr/bin/vdrsync-buffer.pl",
+    "vdrsync.pl"      =>"/usr/bin/vdrsync.pl",
 );
 
 
